packages/website/src/features/Deploy/PublishUtility.tsx

```diff
@@ -72,15 +76,14 @@ export default function PublishUtility(props: {
       );
 
       const targetRegistry = new OnChainRegistry({
```

Comment:
its probably also necessary to supply `provider` here along with the signer because of the way viem works